Orange/data/table.py

```diff
@@ -523,13 +525,51 @@ def from_file(filename, wrapper=None):
 
     @classmethod
     def from_url(cls, url):
-        name = os.path.basename(urllib.parse.urlparse(url)[2])
-        f = tempfile.NamedTemporaryFile(suffix=name, delete=False)
-        fname = f.name
-        f.close()
-        urllib.request.urlretrieve(url, fname)
-        data = cls.from_file(f.name)
-        os.remove(fname)
+        # Resolve (potential) redirects to a final URL
+        response = urlopen(url, timeout=10)
+        url = response.url
+        response.close()
+
+        def url_googlesheets(url):
+            match = re.match(r'(?:https?://)?(?:www\.)?'
+                             'docs\.google\.com/spreadsheets/d/'
+                             '(?P<workbook_id>[-\w_]+)'
+                             '(?:/.*?gid=(?P<sheet_id>\d+).*|.*)?',
+                             url, re.IGNORECASE)
+            try:
+                workbook, sheet = match.group('workbook_id'), match.group('sheet_id')
+                if not workbook: raise ValueError
+            except (AttributeError, ValueError):
+                raise ValueError
+            url = 'https://docs.google.com/spreadsheets/d/{}/export?format=tsv'.format(workbook)
+            if sheet: url += '&gid=' + sheet
+            return url
+
+        URL_TRIMMERS = (
+            url_googlesheets,
+        )
+        for trim in URL_TRIMMERS:
```

Comment:
Do you find it particularly hindering readability in this case or is it only about PEP8-compliance _per se_? Do you insist?
```
$ grep -PRnA1 --exclude-dir='pydoc_data' --include='*.py' '^[^#]+?\b(try|else|(el)?if\s+[^:]+):\s+[^#]+$' /usr/lib/python3.4/
/usr/lib/python3.4/tempfile.py:260:        if dirname: dirlist.append(dirname)
/usr/lib/python3.4/tempfile.py-261-
--
/usr/lib/python3.4/tempfile.py:670:        if self._rolled: return
/usr/lib/python3.4/tempfile.py-671-        max_size = self._max_size
--
/usr/lib/python3.4/tempfile.py:676:        if self._rolled: return
/usr/lib/python3.4/tempfile.py-677-        file = self._file
--
/usr/lib/python3.4/bisect.py:18:        if x < a[mid]: hi = mid
/usr/lib/python3.4/bisect.py:19:        else: lo = mid+1
/usr/lib/python3.4/bisect.py-20-    a.insert(lo, x)
--
/usr/lib/python3.4/bisect.py:41:        if x < a[mid]: hi = mid
/usr/lib/python3.4/bisect.py:42:        else: lo = mid+1
/usr/lib/python3.4/bisect.py-43-    return lo
--
/usr/lib/python3.4/bisect.py:62:        if a[mid] < x: lo = mid+1
/usr/lib/python3.4/bisect.py:63:        else: hi = mid
/usr/lib/python3.4/bisect.py-64-    a.insert(lo, x)
--
/usr/lib/python3.4/bisect.py:84:        if a[mid] < x: lo = mid+1
/usr/lib/python3.4/bisect.py:85:        else: hi = mid
/usr/lib/python3.4/bisect.py-86-    return lo
--
/usr/lib/python3.4/cgi.py:168:            if qs: qs = qs + '&'
/usr/lib/python3.4/cgi.py-169-            qs = qs + environ['QUERY_STRING']
--
/usr/lib/python3.4/cgi.py:171:            if qs: qs = qs + '&'
/usr/lib/python3.4/cgi.py-172-            qs = qs + sys.argv[1]
--
/usr/lib/python3.4/cgi.py:596:            if item.name == key: found.append(item)
/usr/lib/python3.4/cgi.py-597-        if not found:
--
/usr/lib/python3.4/telnetlib.py:613:                if not re: import re
/usr/lib/python3.4/telnetlib.py-614-                list[i] = re.compile(list[i])
--
/usr/lib/python3.4/genericpath.py:71:    if not m: return ''
/usr/lib/python3.4/genericpath.py-72-    s1 = min(m)
--
/usr/lib/python3.4/macpath.py:75:    if colon not in s: return s[:0], s
/usr/lib/python3.4/macpath.py-76-    col = 0
--
/usr/lib/python3.4/webbrowser.py:192:            if opt: raise_opt = [opt]
/usr/lib/python3.4/webbrowser.py-193-
--
/usr/lib/python3.4/webbrowser.py:639:        if o == '-n': new_win = 1
/usr/lib/python3.4/webbrowser.py:640:        elif o == '-t': new_win = 2
/usr/lib/python3.4/webbrowser.py-641-    if len(args) != 1:
--
/usr/lib/python3.4/pydoc.py:196:    if name.startswith('__') and name.endswith('__'): return 1
/usr/lib/python3.4/pydoc.py-197-    # Namedtuples have public fields and methods with a single leading underscore
--
/usr/lib/python3.4/pydoc.py:229:        if not line: break
/usr/lib/python3.4/pydoc.py-230-    line = line.strip()
--
/usr/lib/python3.4/pydoc.py:237:            if not line: break
/usr/lib/python3.4/pydoc.py-238-        result = line.split('"""')[0].strip()
/usr/lib/python3.4/pydoc.py:239:    else: result = None
/usr/lib/python3.4/pydoc.py-240-    return result
--
/usr/lib/python3.4/pydoc.py:353:        try: module = getattr(module, part)
/usr/lib/python3.4/pydoc.py-354-        except AttributeError: return None
--
/usr/lib/python3.4/pydoc.py:372:        if inspect.isgetsetdescriptor(object): return self.docdata(*args)
/usr/lib/python3.4/pydoc.py:373:        if inspect.ismemberdescriptor(object): return self.docdata(*args)
/usr/lib/python3.4/pydoc.py-374-        try:
/usr/lib/python3.4/pydoc.py:375:            if inspect.ismodule(object): return self.docmodule(*args)
/usr/lib/python3.4/pydoc.py:376:            if inspect.isclass(object): return self.docclass(*args)
/usr/lib/python3.4/pydoc.py:377:            if inspect.isroutine(object): return self.docroutine(*args)
/usr/lib/python3.4/pydoc.py-378-        except AttributeError:
--
/usr/lib/python3.4/pydoc.py:380:        if isinstance(object, property): return self.docproperty(*args)
/usr/lib/python3.4/pydoc.py-381-        return self.docother(*args)
--
/usr/lib/python3.4/pydoc.py:590:            if not match: break
/usr/lib/python3.4/pydoc.py-591-            start, end = match.span()
--
/usr/lib/python3.4/pydoc.py:706:                    if inspect.isfunction(value): fdict[value] = fdict[key]
/usr/lib/python3.4/pydoc.py-707-        data = []
--
/usr/lib/python3.4/pydoc.py:1004:        if shadowed is None: shadowed = {}
/usr/lib/python3.4/pydoc.py-1005-        for importer, name, ispkg in pkgutil.iter_modules([dir]):
--
/usr/lib/python3.4/pydoc.py:1064:        if not text: return ''
/usr/lib/python3.4/pydoc.py-1065-        lines = [prefix + line for line in text.split('\n')]
/usr/lib/python3.4/pydoc.py:1066:        if lines: lines[-1] = lines[-1].rstrip()
/usr/lib/python3.4/pydoc.py-1067-        return '\n'.join(lines)
--
/usr/lib/python3.4/pydoc.py:1399:            if chop < 0: repr = repr[:chop] + '...'
/usr/lib/python3.4/pydoc.py-1400-        line = (name and self.bold(name) + ' = ' or '') + repr
--
/usr/lib/python3.4/pydoc.py:1508:                if r < 0: r = 0
/usr/lib/python3.4/pydoc.py-1509-            sys.stdout.write('\n' + '\n'.join(lines[r:r+inc]) + '\n')
--
/usr/lib/python3.4/pydoc.py:1553:        if nextmodule: module, n = nextmodule, n + 1
/usr/lib/python3.4/pydoc.py:1554:        else: break
/usr/lib/python3.4/pydoc.py-1555-    if module:
--
/usr/lib/python3.4/pydoc.py:1633:    if done is None: done = {}
/usr/lib/python3.4/pydoc.py-1634-    for importer, modname, ispkg in pkgutil.walk_packages([dir], pkgpath):
--
/usr/lib/python3.4/pydoc.py:1842:                if not request: break
/usr/lib/python3.4/pydoc.py-1843-            except (KeyboardInterrupt, EOFError):
--
/usr/lib/python3.4/pydoc.py:1846:            if request.lower() in ('q', 'quit'): break
/usr/lib/python3.4/pydoc.py-1847-            self.help(request)
--
/usr/lib/python3.4/pydoc.py:1861:            if request == 'help': self.intro()
/usr/lib/python3.4/pydoc.py:1862:            elif request == 'keywords': self.listkeywords()
/usr/lib/python3.4/pydoc.py:1863:            elif request == 'symbols': self.listsymbols()
/usr/lib/python3.4/pydoc.py:1864:            elif request == 'topics': self.listtopics()
/usr/lib/python3.4/pydoc.py:1865:            elif request == 'modules': self.listmodules()
/usr/lib/python3.4/pydoc.py-1866-            elif request[:8] == 'modules ':
--
/usr/lib/python3.4/pydoc.py:1868:            elif request in self.symbols: self.showsymbol(request)
/usr/lib/python3.4/pydoc.py-1869-            elif request in ['True', 'False', 'None']:
--
/usr/lib/python3.4/pydoc.py:1872:            elif request in self.keywords: self.showtopic(request)
/usr/lib/python3.4/pydoc.py:1873:            elif request in self.topics: self.showtopic(request)
/usr/lib/python3.4/pydoc.py:1874:            elif request: doc(request, 'Help on %s:', output=self._output)
/usr/lib/python3.4/pydoc.py:1875:        elif isinstance(request, Helper): self()
/usr/lib/python3.4/pydoc.py:1876:        else: doc(request, 'Help on %s:', output=self._output)
/usr/lib/python3.4/pydoc.py-1877-        self.output.write('\n')
--
/usr/lib/python3.4/pydoc.py:2028:        if key: key = key.lower()
/usr/lib/python3.4/pydoc.py-2029-        self.quit = False
--
/usr/lib/python3.4/pydoc.py:2578:        if not args: raise BadUsage
/usr/lib/python3.4/pydoc.py-2579-        for arg in args:
--
/usr/lib/python3.4/test/support/__init__.py:973:        if os.path.exists(fn): return fn
/usr/lib/python3.4/test/support/__init__.py-974-    return filename
--
/usr/lib/python3.4/test/support/__init__.py:1758:            if not verbose: err += "; run in verbose mode for details"
/usr/lib/python3.4/test/support/__init__.py-1759-        raise TestFailed(err)
--
/usr/lib/python3.4/test/pystone.py:259:    if EnumLoc == Ident3: return TRUE
/usr/lib/python3.4/test/pystone.py-260-    return FALSE
--
/usr/lib/python3.4/test/pystone.py:272:        try: loops = int(sys.argv[1])
/usr/lib/python3.4/test/pystone.py-273-        except ValueError:
--
/usr/lib/python3.4/threading.py:651:        if not self._cond.wait_for(lambda : self._state != 0, timeout):
/usr/lib/python3.4/threading.py-652-            #timed out.  Break the barrier
--
/usr/lib/python3.4/mailcap.py:65:        if not line: break
/usr/lib/python3.4/mailcap.py-66-        # Ignore comments and blank lines
--
/usr/lib/python3.4/mailcap.py:73:            if not nextline: nextline = '\n'
/usr/lib/python3.4/mailcap.py-74-            line = line[:-2] + nextline
--
/usr/lib/python3.4/mailcap.py:239:    if not caps: caps = getcaps()
/usr/lib/python3.4/mailcap.py-240-    print("Mailcap entries:")
--
/usr/lib/python3.4/pdb.py:716:        if id == '': return failed
/usr/lib/python3.4/pdb.py-717-        parts = id.split('.')
--
/usr/lib/python3.4/pdb.py:1127:        if co.co_flags & 4: n = n+1
/usr/lib/python3.4/pdb.py:1128:        if co.co_flags & 8: n = n+1
/usr/lib/python3.4/pdb.py-1129-        for i in range(n):
--
/usr/lib/python3.4/pdb.py:1414:        if len(args) == 0: return
/usr/lib/python3.4/pdb.py-1415-        if args[0] in self.aliases:
--
/usr/lib/python3.4/base64.py:580:        if o == '-e': func = encode
/usr/lib/python3.4/base64.py:581:        if o == '-d': func = decode
/usr/lib/python3.4/base64.py:582:        if o == '-u': func = decode
/usr/lib/python3.4/base64.py:583:        if o == '-t': test(); return
/usr/lib/python3.4/base64.py-584-    if args and args[0] != '-':
--
/usr/lib/python3.4/fileinput.py:295:            try: os.unlink(backupfilename)
/usr/lib/python3.4/fileinput.py-296-            except OSError: pass
--
/usr/lib/python3.4/fileinput.py:419:        if o == '-i': inplace = True
/usr/lib/python3.4/fileinput.py:420:        if o == '-b': backup = a
/usr/lib/python3.4/fileinput.py-421-    for line in input(args, inplace=inplace, backup=backup):
--
/usr/lib/python3.4/http/client.py:1104:                if self.debuglevel > 0: print("Cannot stat!!")
/usr/lib/python3.4/http/client.py-1105-
--
/usr/lib/python3.4/http/server.py:815:            if word in (os.curdir, os.pardir): continue
/usr/lib/python3.4/http/server.py-816-            path = os.path.join(path, word)
--
/usr/lib/python3.4/http/cookiejar.py:162:    if hr is None: hr = 0
/usr/lib/python3.4/http/cookiejar.py:163:    if min is None: min = 0
/usr/lib/python3.4/http/cookiejar.py:164:    if sec is None: sec = 0
/usr/lib/python3.4/http/cookiejar.py-165-
--
/usr/lib/python3.4/http/cookiejar.py:180:            if m > 0: yr = yr + 100
/usr/lib/python3.4/http/cookiejar.py:181:            else: yr = yr - 100
/usr/lib/python3.4/http/cookiejar.py-182-
--
/usr/lib/python3.4/http/cookiejar.py:407:                if pairs: result.append(pairs)
/usr/lib/python3.4/http/cookiejar.py-408-                pairs = []
--
/usr/lib/python3.4/http/cookiejar.py:416:        if pairs: result.append(pairs)
/usr/lib/python3.4/http/cookiejar.py-417-    return result
--
/usr/lib/python3.4/http/cookiejar.py:442:        if attr: headers.append("; ".join(attr))
/usr/lib/python3.4/http/cookiejar.py-443-    return ", ".join(headers)
--
/usr/lib/python3.4/http/cookiejar.py:477:            if param == "": continue
/usr/lib/python3.4/http/cookiejar.py-478-            if "=" not in param:
--
/usr/lib/python3.4/http/cookiejar.py:744:        if version is not None: version = int(version)
/usr/lib/python3.4/http/cookiejar.py:745:        if expires is not None: expires = int(expires)
/usr/lib/python3.4/http/cookiejar.py-746-        if port is None and port_specified is True:
--
/usr/lib/python3.4/http/cookiejar.py:781:        if now is None: now = time.time()
/usr/lib/python3.4/http/cookiejar.py-782-        if (self.expires is not None) and (self.expires <= now):
--
/usr/lib/python3.4/http/cookiejar.py:787:        if self.port is None: p = ""
/usr/lib/python3.4/http/cookiejar.py:788:        else: p = ":"+self.port
/usr/lib/python3.4/http/cookiejar.py-789-        limit = self.domain + p + self.path
--
/usr/lib/python3.4/http/cookiejar.py:1482:            if len(path) == 0: path = "/"
/usr/lib/python3.4/http/cookiejar.py-1483-
--
/usr/lib/python3.4/http/cookiejar.py:1543:            if cookie: cookies.append(cookie)
/usr/lib/python3.4/http/cookiejar.py-1544-        return cookies
--
/usr/lib/python3.4/http/cookiejar.py:1630:            if cookie.domain not in c: c[cookie.domain] = {}
/usr/lib/python3.4/http/cookiejar.py-1631-            c2 = c[cookie.domain]
/usr/lib/python3.4/http/cookiejar.py:1632:            if cookie.path not in c2: c2[cookie.path] = {}
/usr/lib/python3.4/http/cookiejar.py-1633-            c3 = c2[cookie.path]
--
/usr/lib/python3.4/http/cookiejar.py:1761:            if self.filename is not None: filename = self.filename
/usr/lib/python3.4/http/cookiejar.py:1762:            else: raise ValueError(MISSING_FILENAME_TEXT)
/usr/lib/python3.4/http/cookiejar.py-1763-
--
/usr/lib/python3.4/http/cookiejar.py:1776:            if self.filename is not None: filename = self.filename
/usr/lib/python3.4/http/cookiejar.py:1777:            else: raise ValueError(MISSING_FILENAME_TEXT)
/usr/lib/python3.4/http/cookiejar.py-1778-
--
/usr/lib/python3.4/http/cookiejar.py:1803:    if cookie.port is not None: h.append(("port", cookie.port))
/usr/lib/python3.4/http/cookiejar.py:1804:    if cookie.path_specified: h.append(("path_spec", None))
/usr/lib/python3.4/http/cookiejar.py:1805:    if cookie.port_specified: h.append(("port_spec", None))
/usr/lib/python3.4/http/cookiejar.py:1806:    if cookie.domain_initial_dot: h.append(("domain_dot", None))
/usr/lib/python3.4/http/cookiejar.py:1807:    if cookie.secure: h.append(("secure", None))
/usr/lib/python3.4/http/cookiejar.py:1808:    if cookie.expires: h.append(("expires",
/usr/lib/python3.4/http/cookiejar.py-1809-                               time2isoz(float(cookie.expires))))
/usr/lib/python3.4/http/cookiejar.py:1810:    if cookie.discard: h.append(("discard", None))
/usr/lib/python3.4/http/cookiejar.py:1811:    if cookie.comment: h.append(("comment", cookie.comment))
/usr/lib/python3.4/http/cookiejar.py:1812:    if cookie.comment_url: h.append(("commenturl", cookie.comment_url))
/usr/lib/python3.4/http/cookiejar.py-1813-
--
/usr/lib/python3.4/http/cookiejar.py:1853:            if self.filename is not None: filename = self.filename
/usr/lib/python3.4/http/cookiejar.py:1854:            else: raise ValueError(MISSING_FILENAME_TEXT)
/usr/lib/python3.4/http/cookiejar.py-1855-
--
/usr/lib/python3.4/http/cookiejar.py:1883:                if line == "": break
/usr/lib/python3.4/http/cookiejar.py-1884-                if not line.startswith(header):
--
/usr/lib/python3.4/http/cookiejar.py:1903:                            if v is None: v = True
/usr/lib/python3.4/http/cookiejar.py-1904-                            standard[k] = v
--
/usr/lib/python3.4/http/cookiejar.py:1994:                if line == "": break
/usr/lib/python3.4/http/cookiejar.py-1995-
--
/usr/lib/python3.4/http/cookiejar.py:1997:                if line.endswith("\n"): line = line[:-1]
/usr/lib/python3.4/http/cookiejar.py-1998-
--
/usr/lib/python3.4/http/cookiejar.py:2049:            if self.filename is not None: filename = self.filename
/usr/lib/python3.4/http/cookiejar.py:2050:            else: raise ValueError(MISSING_FILENAME_TEXT)
/usr/lib/python3.4/http/cookiejar.py-2051-
--
/usr/lib/python3.4/http/cookiejar.py:2060:                if cookie.secure: secure = "TRUE"
/usr/lib/python3.4/http/cookiejar.py:2061:                else: secure = "FALSE"
/usr/lib/python3.4/http/cookiejar.py:2062:                if cookie.domain.startswith("."): initial_dot = "TRUE"
/usr/lib/python3.4/http/cookiejar.py:2063:                else: initial_dot = "FALSE"
/usr/lib/python3.4/http/cookiejar.py-2064-                if cookie.expires is not None:
--
/usr/lib/python3.4/encodings/punycode.py:73:    if res < 1: return 1
/usr/lib/python3.4/encodings/punycode.py:74:    if res > 26: return 26
/usr/lib/python3.4/encodings/punycode.py-75-    return res
--
/usr/lib/python3.4/tkinter/tix.py:400:        if kw and cnf: cnf = _cnfmerge((cnf, kw))
/usr/lib/python3.4/tkinter/tix.py:401:        elif kw: cnf = kw
/usr/lib/python3.4/tkinter/tix.py-402-        options = ()
--
/usr/lib/python3.4/tkinter/tix.py:480:        if not master and 'refwindow' in cnf: master=cnf['refwindow']
/usr/lib/python3.4/tkinter/tix.py:481:        elif not master and 'refwindow' in kw:  master= kw['refwindow']
/usr/lib/python3.4/tkinter/tix.py:482:        elif not master: raise RuntimeError("Too early to create display style: no root window")
/usr/lib/python3.4/tkinter/tix.py-483-        self.tk = master.tk
--
/usr/lib/python3.4/tkinter/dialog.py:23:        try: Widget.destroy(self)
/usr/lib/python3.4/tkinter/dialog.py-24-        except TclError: pass
--
/usr/lib/python3.4/tkinter/messagebox.py:68:    if _icon and "icon" not in options:    options["icon"] = _icon
/usr/lib/python3.4/tkinter/messagebox.py:69:    if _type and "type" not in options:    options["type"] = _type
/usr/lib/python3.4/tkinter/messagebox.py:70:    if title:   options["title"] = title
/usr/lib/python3.4/tkinter/messagebox.py:71:    if message: options["message"] = message
/usr/lib/python3.4/tkinter/messagebox.py-72-    res = Message(**options).show()
--
/usr/lib/python3.4/tkinter/__init__.py:535:        if name == 'none' or not name: return None
/usr/lib/python3.4/tkinter/__init__.py-536-        return self._nametowidget(name)
--
/usr/lib/python3.4/tkinter/__init__.py:543:        if name == 'none' or not name: return None
/usr/lib/python3.4/tkinter/__init__.py-544-        return self._nametowidget(name)
--
/usr/lib/python3.4/tkinter/__init__.py:549:        if name == 'none' or not name: return None
/usr/lib/python3.4/tkinter/__init__.py-550-        return self._nametowidget(name)
--
/usr/lib/python3.4/tkinter/__init__.py:565:        if not name: return None
/usr/lib/python3.4/tkinter/__init__.py-566-        return self._nametowidget(name)
--
/usr/lib/python3.4/tkinter/__init__.py:570:        if not name: return None
/usr/lib/python3.4/tkinter/__init__.py-571-        return self._nametowidget(name)
--
/usr/lib/python3.4/tkinter/__init__.py:648:        if 'displayof' not in kw: kw['displayof'] = self._w
/usr/lib/python3.4/tkinter/__init__.py-649-        self.tk.call(('clipboard', 'clear') + self._options(kw))
--
/usr/lib/python3.4/tkinter/__init__.py:656:        if 'displayof' not in kw: kw['displayof'] = self._w
/usr/lib/python3.4/tkinter/__init__.py-657-        self.tk.call(('clipboard', 'append') + self._options(kw)
--
/usr/lib/python3.4/tkinter/__init__.py:664:        if not name: return None
/usr/lib/python3.4/tkinter/__init__.py-665-        return self._nametowidget(name)
--
/usr/lib/python3.4/tkinter/__init__.py:686:        if status == 'none': status = None
/usr/lib/python3.4/tkinter/__init__.py-687-        return status
--
/usr/lib/python3.4/tkinter/__init__.py:714:        if 'displayof' not in kw: kw['displayof'] = self._w
/usr/lib/python3.4/tkinter/__init__.py-715-        self.tk.call(('selection', 'clear') + self._options(kw))
--
/usr/lib/python3.4/tkinter/__init__.py:725:        if 'displayof' not in kw: kw['displayof'] = self._w
/usr/lib/python3.4/tkinter/__init__.py-726-        if 'type' not in kw and self._windowingsystem == 'x11':
--
/usr/lib/python3.4/tkinter/__init__.py:762:        if 'displayof' not in kw: kw['displayof'] = self._w
/usr/lib/python3.4/tkinter/__init__.py-763-        name = self.tk.call(('selection', 'own') + self._options(kw))
/usr/lib/python3.4/tkinter/__init__.py:764:        if not name: return None
/usr/lib/python3.4/tkinter/__init__.py-765-        return self._nametowidget(name)
--
/usr/lib/python3.4/tkinter/__init__.py:817:        if not name: return None
/usr/lib/python3.4/tkinter/__init__.py-818-        return self._nametowidget(name)
--
/usr/lib/python3.4/tkinter/__init__.py:1164:                if k[-1] == '_': k = k[:-1]
/usr/lib/python3.4/tkinter/__init__.py-1165-                if callable(v):
--
/usr/lib/python3.4/tkinter/__init__.py:1230:        if len(args) != len(self._subst_format): return args
/usr/lib/python3.4/tkinter/__init__.py-1231-        getboolean = self.tk.getboolean
--
/usr/lib/python3.4/tkinter/__init__.py:1259:        try: e.focus = getboolean(f)
/usr/lib/python3.4/tkinter/__init__.py-1260-        except TclError: pass
--
/usr/lib/python3.4/tkinter/__init__.py:1269:        try: e.send_event = getboolean(E)
/usr/lib/python3.4/tkinter/__init__.py-1270-        except TclError: pass
--
/usr/lib/python3.4/tkinter/__init__.py:1906:        if 'HOME' in os.environ: home = os.environ['HOME']
/usr/lib/python3.4/tkinter/__init__.py:1907:        else: home = os.curdir
/usr/lib/python3.4/tkinter/__init__.py-1908-        class_tcl = os.path.join(home, '.%s.tcl' % className)
--
/usr/lib/python3.4/tkinter/__init__.py:2164:                if wmkey[-1] == '_': opt = '-'+wmkey[:-1]
/usr/lib/python3.4/tkinter/__init__.py:2165:                else: opt = '-'+wmkey
/usr/lib/python3.4/tkinter/__init__.py-2166-                extra = extra + (opt, val)
--
/usr/lib/python3.4/tkinter/__init__.py:2643:        if i == 'none': return None
/usr/lib/python3.4/tkinter/__init__.py-2644-        return getint(i)
--
/usr/lib/python3.4/tkinter/__init__.py:2805:        if i == 'none': return None
/usr/lib/python3.4/tkinter/__init__.py-2806-        return getint(i)
--
/usr/lib/python3.4/tkinter/__init__.py:3045:                if kw[key]: args.append("-" + key)
/usr/lib/python3.4/tkinter/__init__.py-3046-            args.append(index1)
--
/usr/lib/python3.4/tkinter/__init__.py:3198:        if forwards: args.append('-forwards')
/usr/lib/python3.4/tkinter/__init__.py:3199:        if backwards: args.append('-backwards')
/usr/lib/python3.4/tkinter/__init__.py:3200:        if exact: args.append('-exact')
/usr/lib/python3.4/tkinter/__init__.py:3201:        if regexp: args.append('-regexp')
/usr/lib/python3.4/tkinter/__init__.py:3202:        if nocase: args.append('-nocase')
/usr/lib/python3.4/tkinter/__init__.py:3203:        if elide: args.append('-elide')
/usr/lib/python3.4/tkinter/__init__.py:3204:        if count: args.append('-count'); args.append(count)
/usr/lib/python3.4/tkinter/__init__.py:3205:        if pattern and pattern[0] == '-': args.append('--')
/usr/lib/python3.4/tkinter/__init__.py-3206-        args.append(pattern)
--
/usr/lib/python3.4/tkinter/__init__.py:3208:        if stopindex: args.append(stopindex)
/usr/lib/python3.4/tkinter/__init__.py-3209-        return str(self.tk.call(tuple(args)))
--
/usr/lib/python3.4/tkinter/__init__.py:3367:            if name[0] == '-': name = '_' + name[1:]
/usr/lib/python3.4/tkinter/__init__.py:3368:        if kw and cnf: cnf = _cnfmerge((cnf, kw))
/usr/lib/python3.4/tkinter/__init__.py:3369:        elif kw: cnf = kw
/usr/lib/python3.4/tkinter/__init__.py-3370-        options = ()
--
/usr/lib/python3.4/tkinter/__init__.py:3394:                if k[-1] == '_': k = k[:-1]
/usr/lib/python3.4/tkinter/__init__.py-3395-                if callable(v):
--
/usr/lib/python3.4/tkinter/__init__.py:3439:        if y=='': y=x
/usr/lib/python3.4/tkinter/__init__.py-3440-        self.tk.call(destImage, 'copy', self.name, '-zoom',x,y)
--
/usr/lib/python3.4/tkinter/__init__.py:3446:        if y=='': y=x
/usr/lib/python3.4/tkinter/__init__.py-3447-        self.tk.call(destImage, 'copy', self.name, '-subsample',x,y)
--
/usr/lib/python3.4/tkinter/filedialog.py:34:        if fname is None: ...canceled...
/usr/lib/python3.4/tkinter/filedialog.py:35:        else: ...open file...
/usr/lib/python3.4/tkinter/filedialog.py-36-
--
/usr/lib/python3.4/tkinter/filedialog.py:52:        if title is None: title = self.title
/usr/lib/python3.4/tkinter/filedialog.py-53-        self.master = master
--
/usr/lib/python3.4/tkinter/filedialog.py:190:        if tail == os.curdir: tail = ''
/usr/lib/python3.4/tkinter/filedialog.py-191-        self.set_selection(tail)
--
/usr/lib/python3.4/modulefinder.py:198:            if i < 0: i = len(tail)
/usr/lib/python3.4/modulefinder.py-199-            head, tail = tail[:i], tail[i+1:]
--
/usr/lib/python3.4/xmlrpc/server.py:717:            if not match: break
/usr/lib/python3.4/xmlrpc/server.py-718-            start, end = match.span()
--
/usr/lib/python3.4/trace.py:505:        if globals is None: globals = {}
/usr/lib/python3.4/trace.py:506:        if locals is None: locals = {}
/usr/lib/python3.4/trace.py-507-        if not self.donothing:
--
/usr/lib/python3.4/keyword.py:60:    if len(args) > 1: optfile = args[1]
/usr/lib/python3.4/keyword.py:61:    else: optfile = "Lib/keyword.py"
/usr/lib/python3.4/keyword.py-62-
--
/usr/lib/python3.4/optparse.py:712:                    "callback_args, if supplied, must be a tuple: not %r"
/usr/lib/python3.4/optparse.py-713-                    % self.callback_args, self)
--
/usr/lib/python3.4/optparse.py:717:                    "callback_kwargs, if supplied, must be a dict: not %r"
/usr/lib/python3.4/optparse.py-718-                    % self.callback_kwargs, self)
--
/usr/lib/python3.4/optparse.py:1269:        elif usage.lower().startswith("usage: "):
/usr/lib/python3.4/optparse.py-1270-            self.usage = usage[7:]
--
/usr/lib/python3.4/inspect.py:709:            if pat.match(lines[lnum]): break
/usr/lib/python3.4/inspect.py-710-            lnum = lnum - 1
--
/usr/lib/python3.4/inspect.py:751:                    if end < 0: break
/usr/lib/python3.4/inspect.py-752-                    comment = lines[end].expandtabs().lstrip()
--
/usr/lib/python3.4/inspect.py:821:    if ismodule(object): return lines, 0
/usr/lib/python3.4/inspect.py:822:    else: return getblock(lines[lnum:]), lnum + 1
/usr/lib/python3.4/inspect.py-823-
--
/usr/lib/python3.4/inspect.py:862:                if unique and parent in classes: break
/usr/lib/python3.4/inspect.py-863-        elif c not in roots:
--
/usr/lib/python3.4/wsgiref/validate.py:36:  - Warns if the REQUEST_METHOD is not known (@@: probably too
/usr/lib/python3.4/wsgiref/validate.py-37-    restrictive).
--
/usr/lib/python3.4/gettext.py:144:            if i & COMPONENT_TERRITORY: val += territory
/usr/lib/python3.4/gettext.py:145:            if i & COMPONENT_CODESET:   val += codeset
/usr/lib/python3.4/gettext.py:146:            if i & COMPONENT_MODIFIER:  val += modifier
/usr/lib/python3.4/gettext.py-147-            ret.append(val)
--
/usr/lib/python3.4/ftplib.py:197:        if self.debugging: print('*cmd*', self.sanitize(line))
/usr/lib/python3.4/ftplib.py-198-        self.putline(line)
--
/usr/lib/python3.4/ftplib.py:537:                    if buf[-1] in B_CRLF: buf = buf[:-1]
/usr/lib/python3.4/ftplib.py-538-                    buf = buf + B_CRLF
--
/usr/lib/python3.4/ftplib.py:859:    if left < 0: raise error_proto(resp)
/usr/lib/python3.4/ftplib.py-860-    right = resp.find(')', left + 1)
--
/usr/lib/python3.4/ctypes/__init__.py:532:    if sizeof(kind) == 2: c_int16 = kind
/usr/lib/python3.4/ctypes/__init__.py:533:    elif sizeof(kind) == 4: c_int32 = kind
/usr/lib/python3.4/ctypes/__init__.py:534:    elif sizeof(kind) == 8: c_int64 = kind
/usr/lib/python3.4/ctypes/__init__.py-535-for kind in [c_ushort, c_uint, c_ulong, c_ulonglong]:
/usr/lib/python3.4/ctypes/__init__.py:536:    if sizeof(kind) == 2: c_uint16 = kind
/usr/lib/python3.4/ctypes/__init__.py:537:    elif sizeof(kind) == 4: c_uint32 = kind
/usr/lib/python3.4/ctypes/__init__.py:538:    elif sizeof(kind) == 8: c_uint64 = kind
/usr/lib/python3.4/ctypes/__init__.py-539-del(kind)
--
/usr/lib/python3.4/quopri.py:131:        if not line: break
/usr/lib/python3.4/quopri.py-132-        i, n = 0, len(line)
--
/usr/lib/python3.4/quopri.py:210:        if o == '-t': tabs = 1
/usr/lib/python3.4/quopri.py:211:        if o == '-d': deco = 1
/usr/lib/python3.4/quopri.py-212-    if tabs and deco:
--
/usr/lib/python3.4/quopri.py:216:    if not args: args = ['-']
/usr/lib/python3.4/quopri.py-217-    sts = 0
--
/usr/lib/python3.4/urllib/parse.py:886:    if match: return match.group(1, 2)
/usr/lib/python3.4/urllib/parse.py-887-    return None, host
--
/usr/lib/python3.4/urllib/parse.py:897:    if match: return match.group(1, 2)
/usr/lib/python3.4/urllib/parse.py-898-    return user, None
--
/usr/lib/python3.4/urllib/parse.py:944:    if match: return match.group(1, 2)
/usr/lib/python3.4/urllib/parse.py-945-    return url, None
--
/usr/lib/python3.4/urllib/parse.py:955:    if match: return match.group(1, 2)
/usr/lib/python3.4/urllib/parse.py-956-    return url, None
--
/usr/lib/python3.4/urllib/parse.py:972:    if match: return match.group(1, 2)
/usr/lib/python3.4/urllib/parse.py-973-    return attr, None
--
/usr/lib/python3.4/urllib/request.py:1743:        if not host: raise OSError('http error', 'no host given')
/usr/lib/python3.4/urllib/request.py-1744-
--
/usr/lib/python3.4/urllib/request.py:1812:            if result: return result
/usr/lib/python3.4/urllib/request.py-1813-        return self.http_error_default(url, fp, errcode, errmsg, headers)
--
/usr/lib/python3.4/urllib/request.py:1877:        if not host: raise URLError('ftp error: no host given')
/usr/lib/python3.4/urllib/request.py-1878-        host, port = splitport(host)
--
/usr/lib/python3.4/urllib/request.py:1880:        if user: user, passwd = splitpasswd(user)
/usr/lib/python3.4/urllib/request.py:1881:        else: passwd = None
/usr/lib/python3.4/urllib/request.py-1882-        host = unquote(host)
--
/usr/lib/python3.4/urllib/request.py:1895:        if dirs and not dirs[0]: dirs = dirs[1:]
/usr/lib/python3.4/urllib/request.py:1896:        if dirs and not dirs[0]: dirs[0] = '/'
/usr/lib/python3.4/urllib/request.py-1897-        key = user, host, port, '/'.join(dirs)
--
/usr/lib/python3.4/urllib/request.py:1910:            if not file: type = 'D'
/usr/lib/python3.4/urllib/request.py:1911:            else: type = 'I'
/usr/lib/python3.4/urllib/request.py-1912-            for attr in attrs:
--
/usr/lib/python3.4/urllib/request.py:2102:        if not (user or passwd): return None
/usr/lib/python3.4/urllib/request.py-2103-        proxyhost = "%s:%s@%s" % (quote(user, safe=''),
--
/usr/lib/python3.4/urllib/request.py:2120:        if not (user or passwd): return None
/usr/lib/python3.4/urllib/request.py-2121-        proxyhost = "%s:%s@%s" % (quote(user, safe=''),
--
/usr/lib/python3.4/urllib/request.py:2134:        if not (user or passwd): return None
/usr/lib/python3.4/urllib/request.py-2135-        host = "%s:%s@%s" % (quote(user, safe=''),
--
/usr/lib/python3.4/urllib/request.py:2148:        if not (user or passwd): return None
/usr/lib/python3.4/urllib/request.py-2149-        host = "%s:%s@%s" % (quote(user, safe=''),
--
/usr/lib/python3.4/urllib/request.py:2165:        if user or passwd: self.auth_cache[key] = (user, passwd)
/usr/lib/python3.4/urllib/request.py-2166-        return user, passwd
--
/usr/lib/python3.4/urllib/request.py:2249:        if type in ('d', 'D'): cmd = 'TYPE A'; isdir = 1
/usr/lib/python3.4/urllib/request.py:2250:        else: cmd = 'TYPE ' + type; isdir = 0
/usr/lib/python3.4/urllib/request.py-2251-        try:
--
/usr/lib/python3.4/urllib/request.py:2384:        if not value: continue
/usr/lib/python3.4/urllib/request.py-2385-
--
/usr/lib/python3.4/mimetypes.py:566:        if msg: print(msg)
/usr/lib/python3.4/mimetypes.py-567-        sys.exit(code)
--
/usr/lib/python3.4/mimetypes.py:587:            if not guess: print("I don't know anything about type", gtype)
/usr/lib/python3.4/mimetypes.py:588:            else: print(guess)
/usr/lib/python3.4/mimetypes.py-589-        else:
--
/usr/lib/python3.4/mimetypes.py:591:            if not guess: print("I don't know anything about type", gtype)
/usr/lib/python3.4/mimetypes.py:592:            else: print('type:', guess, 'encoding:', encoding)
--
/usr/lib/python3.4/lib2to3/pytree.py:29:            if type(val) == int: _type_reprs[val] = name
/usr/lib/python3.4/lib2to3/pytree.py-30-    return _type_reprs.setdefault(type_num, type_num)
--
/usr/lib/python3.4/lib2to3/btm_utils.py:254:        if any(rec_test(subpattern, lambda x: type(x) is str)):
/usr/lib/python3.4/lib2to3/btm_utils.py-255-            if any(rec_test(subpattern,
--
/usr/lib/python3.4/lib2to3/fixer_util.py:374:            if n: ret = n
/usr/lib/python3.4/lib2to3/fixer_util.py-375-        elif child.type in (syms.if_stmt, syms.while_stmt):
--
/usr/lib/python3.4/lib2to3/fixer_util.py:377:            if n: ret = n
/usr/lib/python3.4/lib2to3/fixer_util.py-378-        elif child.type == syms.try_stmt:
--
/usr/lib/python3.4/lib2to3/fixer_util.py:387:                        if n: ret = n
/usr/lib/python3.4/lib2to3/fixer_util.py-388-        elif child.type in _def_syms and child.children[1].value == name:
--
/usr/lib/python3.4/lib2to3/btm_matcher.py:167:            if type(val) == int: _type_reprs[val] = name
/usr/lib/python3.4/lib2to3/btm_matcher.py-168-    return _type_reprs.setdefault(type_num, type_num)
--
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:399:            if not line: break
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py-400-            column = 0
--
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:402:                if line[pos] == ' ': column = column + 1
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:403:                elif line[pos] == '\t': column = (column//tabsize + 1)*tabsize
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:404:                elif line[pos] == '\f': column = 0
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:405:                else: break
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py-406-                pos = pos + 1
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:407:            if pos == max: break
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py-408-
--
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:487:                    if initial in '([{': parenlev = parenlev + 1
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:488:                    elif initial in ')]}': parenlev = parenlev - 1
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py-489-                    yield (OP, token, spos, epos, line)
--
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:501:    if len(sys.argv) > 1: tokenize(open(sys.argv[1]).readline)
/usr/lib/python3.4/lib2to3/pgen2/tokenize.py:502:    else: tokenize(sys.stdin.readline)
--
/usr/lib/python3.4/reprlib.py:74:            if n > maxiter:  pieces.append('...')
/usr/lib/python3.4/reprlib.py-75-            s = ', '.join(pieces)
/usr/lib/python3.4/reprlib.py:76:            if n == 1 and trail:  right = trail + right
/usr/lib/python3.4/reprlib.py-77-        return '%s%s%s' % (left, s, right)
--
/usr/lib/python3.4/reprlib.py:103:        if n == 0: return '{}'
/usr/lib/python3.4/reprlib.py:104:        if level <= 0: return '{...}'
/usr/lib/python3.4/reprlib.py-105-        newlevel = level - 1
--
/usr/lib/python3.4/reprlib.py:112:        if n > self.maxdict: pieces.append('...')
/usr/lib/python3.4/reprlib.py-113-        s = ', '.join(pieces)
--
/usr/lib/python3.4/imghdr.py:25:        if f: f.close()
/usr/lib/python3.4/imghdr.py-26-    return None
--
/usr/lib/python3.4/tabnanny.py:124:            if ' ' in file: file = '"' + file + '"'
/usr/lib/python3.4/tabnanny.py:125:            if filename_only: print(file)
/usr/lib/python3.4/tabnanny.py:126:            else: print(file, badline, repr(line))
/usr/lib/python3.4/tabnanny.py-127-        return
--
/usr/lib/python3.4/binhex.py:236:        if not d: break
/usr/lib/python3.4/binhex.py-237-        ofp.write(d)
--
/usr/lib/python3.4/binhex.py:244:        if not d: break
/usr/lib/python3.4/binhex.py-245-        ofp.write_rsrc(d)
--
/usr/lib/python3.4/binhex.py:264:            if self.eof: return decdata
/usr/lib/python3.4/binhex.py-265-            wtd = ((wtd + 2) // 3) * 4
--
/usr/lib/python3.4/binhex.py:456:        if not d: break
/usr/lib/python3.4/binhex.py-457-        ofp.write(d)
--
/usr/lib/python3.4/binhex.py:467:            if not d: break
/usr/lib/python3.4/binhex.py-468-            ofp.write(d)
--
/usr/lib/python3.4/turtle.py:1225:        Arguments (if given): a color string or three numbers
/usr/lib/python3.4/turtle.py-1226-        in the range 0..colormode or a 3-tuple of such numbers.
--
/usr/lib/python3.4/turtle.py:2935:        if t11 is not None: m11 = t11
/usr/lib/python3.4/turtle.py:2936:        if t12 is not None: m12 = t12
/usr/lib/python3.4/turtle.py:2937:        if t21 is not None: m21 = t21
/usr/lib/python3.4/turtle.py:2938:        if t22 is not None: m22 = t22
/usr/lib/python3.4/turtle.py-2939-        if t11 * t22 - t12 * t21 == 0:
--
/usr/lib/python3.4/turtle.py:3005:                if self._resizemode == "noresize": w = 1
/usr/lib/python3.4/turtle.py:3006:                elif self._resizemode == "auto": w = self._pensize
/usr/lib/python3.4/turtle.py:3007:                else: w =self._outlinewidth
/usr/lib/python3.4/turtle.py-3008-                shape = self._polytrafo(self._getshapepoly(tshape))
--
/usr/lib/python3.4/turtle.py:3055:            if self._resizemode == "noresize": w = 1
/usr/lib/python3.4/turtle.py:3056:            elif self._resizemode == "auto": w = self._pensize
/usr/lib/python3.4/turtle.py:3057:            else: w =self._outlinewidth
/usr/lib/python3.4/turtle.py-3058-            shape = self._polytrafo(self._getshapepoly(tshape))
--
/usr/lib/python3.4/stringprep.py:13:    if unicodedata.category(code) != 'Cn': return False
/usr/lib/python3.4/stringprep.py-14-    c = ord(code)
/usr/lib/python3.4/stringprep.py:15:    if 0xFDD0 <= c < 0xFDF0: return False
/usr/lib/python3.4/stringprep.py-16-    return (c & 0xFFFF) not in (0xFFFE, 0xFFFF)
--
/usr/lib/python3.4/stringprep.py:191:    if r is not None: return r
/usr/lib/python3.4/stringprep.py-192-    return code.lower()
--
/usr/lib/python3.4/stringprep.py:223:    if c < 128: return False
/usr/lib/python3.4/stringprep.py:224:    if unicodedata.category(code) == "Cc": return True
/usr/lib/python3.4/stringprep.py-225-    return c in c22_specials
--
/usr/lib/python3.4/stringprep.py:238:    if c < 0xFDD0: return False
/usr/lib/python3.4/stringprep.py:239:    if c < 0xFDF0: return True
/usr/lib/python3.4/stringprep.py-240-    return (ord(code) & 0xFFFF) in (0xFFFE, 0xFFFF)
--
/usr/lib/python3.4/tokenize.py:518:            if not line: break
/usr/lib/python3.4/tokenize.py-519-            column = 0
--
/usr/lib/python3.4/bdb.py:67:            if self.quitting: raise BdbQuit
/usr/lib/python3.4/bdb.py-68-        return self.trace_dispatch
--
/usr/lib/python3.4/bdb.py:83:        if self.quitting: raise BdbQuit
/usr/lib/python3.4/bdb.py-84-        return self.trace_dispatch
--
/usr/lib/python3.4/bdb.py:96:            if self.quitting: raise BdbQuit
/usr/lib/python3.4/bdb.py-97-            # The user issued a 'next' or 'until' command.
--
/usr/lib/python3.4/bdb.py:110:                if self.quitting: raise BdbQuit
/usr/lib/python3.4/bdb.py-111-        # Stop at the StopIteration or GeneratorExit exception when the user
--
/usr/lib/python3.4/bdb.py:119:            if self.quitting: raise BdbQuit
/usr/lib/python3.4/bdb.py-120-
--
/usr/lib/python3.4/bdb.py:647:        if not name: name = '???'
/usr/lib/python3.4/bdb.py-648-        print('+++ call', name, args)
--
/usr/lib/python3.4/bdb.py:652:        if not name: name = '???'
/usr/lib/python3.4/bdb.py-653-        fn = self.canonic(frame.f_code.co_filename)
--
/usr/lib/python3.4/nntplib.py:380:        if self.debugging: print('*welcome*', repr(self.welcome))
/usr/lib/python3.4/nntplib.py-381-        return self.welcome
--
/usr/lib/python3.4/nntplib.py:418:        if self.debugging > 1: print('*put*', repr(line))
/usr/lib/python3.4/nntplib.py-419-        self.file.write(line)
--
/usr/lib/python3.4/nntplib.py:425:        if self.debugging: print('*cmd*', repr(line))
/usr/lib/python3.4/nntplib.py-426-        line = line.encode(self.encoding, self.errors)
--
/usr/lib/python3.4/nntplib.py:438:        if not line: raise EOFError
/usr/lib/python3.4/nntplib.py-439-        if strip_crlf:
--
/usr/lib/python3.4/nntplib.py:451:        if self.debugging: print('*resp*', repr(resp))
/usr/lib/python3.4/nntplib.py-452-        resp = resp.decode(self.encoding, self.errors)
--
/usr/lib/python3.4/cgitb.py:84:        if ttype == tokenize.NEWLINE: break
/usr/lib/python3.4/cgitb.py-85-        if ttype == tokenize.NAME and token not in keyword.kwlist:
--
/usr/lib/python3.4/cgitb.py:134:            try: return linecache.getline(file, lnum[0])
/usr/lib/python3.4/cgitb.py-135-            finally: lnum[0] += 1
--
/usr/lib/python3.4/cgitb.py:154:            if name in done: continue
/usr/lib/python3.4/cgitb.py-155-            done[name] = 1
--
/usr/lib/python3.4/cgitb.py:217:            try: return linecache.getline(file, lnum[0])
/usr/lib/python3.4/cgitb.py-218-            finally: lnum[0] += 1
--
/usr/lib/python3.4/cgitb.py:231:            if name in done: continue
/usr/lib/python3.4/cgitb.py-232-            done[name] = 1
--
/usr/lib/python3.4/cgitb.py:234:                if where == 'global': name = 'global ' + name
/usr/lib/python3.4/cgitb.py:235:                elif where != 'local': name = where + name.split('.')[-1]
/usr/lib/python3.4/cgitb.py-236-                dump.append('%s = %s' % (name, pydoc.text.repr(value)))
--
/usr/lib/python3.4/formatter.py:181:        if not data: return
/usr/lib/python3.4/formatter.py-182-        prespace = data[:1].isspace()
--
/usr/lib/python3.4/formatter.py:201:        if not data: return
/usr/lib/python3.4/formatter.py-202-        if self.softspace:
--
/usr/lib/python3.4/formatter.py:242:            if size is AS_IS: size = csize
/usr/lib/python3.4/formatter.py:243:            if i is AS_IS: i = ci
/usr/lib/python3.4/formatter.py:244:            if b is AS_IS: b = cb
/usr/lib/python3.4/formatter.py:245:            if tt is AS_IS: tt = ctt
/usr/lib/python3.4/formatter.py-246-        font = (size, i, b, tt)
--
/usr/lib/python3.4/formatter.py:410:        if not data: return
/usr/lib/python3.4/formatter.py-411-        atbreak = self.atbreak or data[0].isspace()
--
/usr/lib/python3.4/decimal.py:543:    if ctx is None: ctx = getcontext()
/usr/lib/python3.4/decimal.py-544-    return _ContextManager(ctx)
--
/usr/lib/python3.4/imaplib.py:568:        try: typ, dat = self._simple_command('LOGOUT')
/usr/lib/python3.4/imaplib.py-569-        except: typ, dat = 'NO', ['%s: %s' % sys.exc_info()[:2]]
--
/usr/lib/python3.4/imaplib.py:896:            if arg is None: continue
/usr/lib/python3.4/imaplib.py-897-            if isinstance(arg, str):
--
/usr/lib/python3.4/imaplib.py:1020:            if dat2: dat = dat + b' ' + dat2
/usr/lib/python3.4/imaplib.py-1021-
--
/usr/lib/python3.4/imaplib.py:1161:            if not l: return
/usr/lib/python3.4/imaplib.py-1162-            t = '\n\t\t'
--
/usr/lib/python3.4/imaplib.py:1464:            if not args: args = (stream_command,)
/usr/lib/python3.4/imaplib.py-1465-
/usr/lib/python3.4/imaplib.py:1466:    if not args: args = ('',)
/usr/lib/python3.4/imaplib.py-1467-
--
/usr/lib/python3.4/imaplib.py:1505:        if typ == 'NO': raise dat[0]
/usr/lib/python3.4/imaplib.py-1506-        return dat
--
/usr/lib/python3.4/imaplib.py:1523:            if mo: path = mo.group(1)
/usr/lib/python3.4/imaplib.py:1524:            else: path = ml.split()[-1]
/usr/lib/python3.4/imaplib.py-1525-            run('delete', (path,))
--
/usr/lib/python3.4/imaplib.py:1534:            if not uid: continue
/usr/lib/python3.4/imaplib.py-1535-            run('uid', ('FETCH', '%s' % uid[-1],
--
/usr/lib/python3.4/distutils/msvccompiler.py:315:        if output_dir is None: output_dir = ''
/usr/lib/python3.4/distutils/msvccompiler.py-316-        obj_names = []
--
/usr/lib/python3.4/distutils/fancy_getopt.py:168:                if short: short = short + ':'
/usr/lib/python3.4/distutils/fancy_getopt.py-169-                long = long[0:-1]
--
/usr/lib/python3.4/distutils/sysconfig.py:299:            try: v = int(v)
/usr/lib/python3.4/distutils/sysconfig.py-300-            except ValueError: pass
--
/usr/lib/python3.4/distutils/sysconfig.py:392:                        try: value = int(value)
/usr/lib/python3.4/distutils/sysconfig.py-393-                        except ValueError:
--
/usr/lib/python3.4/distutils/util.py:238:    if _wordchars_re is None: _init_regex()
/usr/lib/python3.4/distutils/util.py-239-
--
/usr/lib/python3.4/distutils/util.py:524:            if not line: continue
/usr/lib/python3.4/distutils/util.py-525-            filelist.process_template_line(line)
--
/usr/lib/python3.4/distutils/util.py:531:        if res[1]: copied.append(outname)
/usr/lib/python3.4/distutils/util.py-532-    run_2to3([fn for fn in copied if fn.lower().endswith('.py')],
--
/usr/lib/python3.4/distutils/dist.py:961:        if the command has already been run).  Specifically: if we have
/usr/lib/python3.4/distutils/dist.py-962-        already created and run the command named by 'command', return
--
/usr/lib/python3.4/distutils/command/bdist_msi.py:223:        if os.path.exists(installer_name): os.unlink(installer_name)
/usr/lib/python3.4/distutils/command/bdist_msi.py-224-
--
/usr/lib/python3.4/distutils/bcppcompiler.py:338:        if output_dir is None: output_dir = ''
/usr/lib/python3.4/distutils/bcppcompiler.py-339-        obj_names = []
--
/usr/lib/python3.4/distutils/msvc9compiler.py:430:        if output_dir is None: output_dir = ''
/usr/lib/python3.4/distutils/msvc9compiler.py-431-        obj_names = []
--
/usr/lib/python3.4/idlelib/EditorWindow.py:1469:        if tabwidth is None: return
/usr/lib/python3.4/idlelib/EditorWindow.py-1470-        for pos in range(len(lines)):
--
/usr/lib/python3.4/idlelib/EditorWindow.py:1481:        if tabwidth is None: return
/usr/lib/python3.4/idlelib/EditorWindow.py-1482-        for pos in range(len(lines)):
--
/usr/lib/python3.4/idlelib/FormatParagraph.py:187:    if m is None: return ""
/usr/lib/python3.4/idlelib/FormatParagraph.py-188-    return m.group(1)
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:78:        if DEBUG: print('tagdefs',self.tagdefs)
/usr/lib/python3.4/idlelib/ColorDelegator.py-79-
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:97:            if DEBUG: print("colorizing already scheduled")
/usr/lib/python3.4/idlelib/ColorDelegator.py-98-            return
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:101:            if DEBUG: print("stop colorizing")
/usr/lib/python3.4/idlelib/ColorDelegator.py-102-        if self.allow_colorizing:
/usr/lib/python3.4/idlelib/ColorDelegator.py:103:            if DEBUG: print("schedule colorizing")
/usr/lib/python3.4/idlelib/ColorDelegator.py-104-            self.after_id = self.after(1, self.recolorize)
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:112:            if DEBUG: print("cancel scheduled recolorizer")
/usr/lib/python3.4/idlelib/ColorDelegator.py-113-            self.after_cancel(after_id)
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:126:            if DEBUG: print("cancel scheduled recolorizer")
/usr/lib/python3.4/idlelib/ColorDelegator.py-127-            self.after_cancel(after_id)
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:129:            if DEBUG: print("stop colorizing")
/usr/lib/python3.4/idlelib/ColorDelegator.py-130-            self.stop_colorizing = True
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:142:            if DEBUG: print("no delegate")
/usr/lib/python3.4/idlelib/ColorDelegator.py-143-            return
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:145:            if DEBUG: print("auto colorizing is off")
/usr/lib/python3.4/idlelib/ColorDelegator.py-146-            return
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:148:            if DEBUG: print("already colorizing")
/usr/lib/python3.4/idlelib/ColorDelegator.py-149-            return
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:153:            if DEBUG: print("colorizing...")
/usr/lib/python3.4/idlelib/ColorDelegator.py-154-            t0 = time.perf_counter()
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:157:            if DEBUG: print("%.3f seconds" % (t1-t0))
/usr/lib/python3.4/idlelib/ColorDelegator.py-158-        finally:
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:161:            if DEBUG: print("reschedule colorizing")
/usr/lib/python3.4/idlelib/ColorDelegator.py-162-            self.after_id = self.after(1, self.recolorize)
--
/usr/lib/python3.4/idlelib/ColorDelegator.py:230:                    if DEBUG: print("colorizing stopped")
/usr/lib/python3.4/idlelib/ColorDelegator.py-231-                    return
--
/usr/lib/python3.4/idlelib/PyShell.py:1376:                if not line: break
/usr/lib/python3.4/idlelib/PyShell.py-1377-                result += line
--
/usr/lib/python3.4/idlelib/PyShell.py:1381:                if not line: break
/usr/lib/python3.4/idlelib/PyShell.py-1382-                result += line
--
/usr/lib/python3.4/xdrlib.py:59:        if x: self.__buf.write(b'\0\0\0\1')
/usr/lib/python3.4/xdrlib.py:60:        else: self.__buf.write(b'\0\0\0\0')
/usr/lib/python3.4/xdrlib.py-61-
--
/usr/lib/python3.4/xdrlib.py:69:        try: self.__buf.write(struct.pack('>f', x))
/usr/lib/python3.4/xdrlib.py-70-        except struct.error as msg:
--
/usr/lib/python3.4/xdrlib.py:74:        try: self.__buf.write(struct.pack('>d', x))
/usr/lib/python3.4/xdrlib.py-75-        except struct.error as msg:
--
/usr/lib/python3.4/xdrlib.py:209:            if x == 0: break
/usr/lib/python3.4/xdrlib.py-210-            if x != 1:
--
/usr/lib/python3.4/html/parser.py:218:            if i == n: break
/usr/lib/python3.4/html/parser.py-219-            startswith = rawdata.startswith
--
/usr/lib/python3.4/doctest.py:469:        if options is None: options = {}
/usr/lib/python3.4/doctest.py-470-        self.options = options
--
/usr/lib/python3.4/doctest.py:1088:        if inspect.ismethod(obj): obj = obj.__func__
/usr/lib/python3.4/doctest.py:1089:        if inspect.isfunction(obj): obj = obj.__code__
/usr/lib/python3.4/doctest.py:1090:        if inspect.istraceback(obj): obj = obj.tb_frame
/usr/lib/python3.4/doctest.py:1091:        if inspect.isframe(obj): obj = obj.f_code
/usr/lib/python3.4/doctest.py-1092-        if inspect.iscode(obj):
--
/usr/lib/python3.4/poplib.py:110:        if self._debugging > 1: print('*put*', repr(line))
/usr/lib/python3.4/poplib.py-111-        self.sock.sendall(line + CRLF)
--
/usr/lib/python3.4/poplib.py:117:        if self._debugging: print('*cmd*', repr(line))
/usr/lib/python3.4/poplib.py-118-        line = bytes(line, self.encoding)
--
/usr/lib/python3.4/poplib.py:131:        if self._debugging > 1: print('*get*', repr(line))
/usr/lib/python3.4/poplib.py:132:        if not line: raise error_proto('-ERR EOF')
/usr/lib/python3.4/poplib.py-133-        octets = len(line)
--
/usr/lib/python3.4/poplib.py:149:        if self._debugging > 1: print('*resp*', repr(resp))
/usr/lib/python3.4/poplib.py-150-        if not resp.startswith(b'+'):
--
/usr/lib/python3.4/poplib.py:222:        if self._debugging: print('*stat*', repr(rets))
/usr/lib/python3.4/poplib.py-223-        numMessages = int(rets[1])
```